ts/features/idpay/details/components/TimelineOperationListItem.tsx

```diff
@@ -236,17 +244,20 @@ const getInstrumentOperationProps = (
       paymentLogoIcon: getLogo(),
       title: getTitle(),
       subtitle,
-      transactionStatus: "failure",
-      badgeText: getBadgeTextByTransactionStatus("failure")
+      transaction: {
+        badge: getBadgePropsByTransactionStatus("failure")
+      }
     };
   }
 
   return {
     paymentLogoIcon: getLogo(),
     title: getTitle(),
     subtitle,
-    transactionStatus: "success",
-    transactionAmount: ""
+    transaction: {
+      amount: "",
+      amountAccessibilityLabel: ""
+    }
```

Comment:
Would it make sense to pass these common props as a single prop object (e.g., `transactionAmounts`)

Reply:
It makes sense to me, addressed into  edd87d2dd775d47e732c551dcb08eb86089fb4d9